Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG GRAFANA_VERSION="latest"
+
+FROM grafana/grafana:${GRAFANA_VERSION}
+
+USER root
+
+ENV GF_PATHS_PLUGINS="/var/lib/grafana-plugins"
+
+RUN mkdir -p "$GF_PATHS_PLUGINS" && \
+    chown -R 472:472 "$GF_PATHS_PLUGINS"
```

Comment:
for readability: is `472` the `grafana` user and group? Then you can directly refer to `grafana:grafana` in `chown`.

Reply:
Yes sorry this is supposed to be grafana:grafana, but I got this error ‘Permission denied. Make sure you have write access to plugindir’ even when the owner of the plugin directory is grafana:grafana.

Reply:
This is resolved now.